app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -24,14 +24,20 @@ def create
         order.update!(status: :paid, placed_at: DateTime.now, payment_processor_fee_cents: balance_transaction.fee)
         order.events.create(description: "Payment Received")
 
+        # TODO: Does this suffice as a feature flag for the time being?
+        # TODO: Need better conditional abstraction here?
+        if marketplace.square_location_id? && Rails.env.development?
+          order.send_to_square_seller_dashboard(balance_transaction)
+        end
```

Comment:
I don't know that we need to hide it behind a feature flag

Reply:
Agree, I'll write a boolean for this.